plugins/scaffolder-backend/src/scaffolder/tasks/types.ts

```diff
@@ -142,6 +143,14 @@ export type TaskStoreCreateTaskOptions = {
   secrets?: TaskSecrets;
 };
 
+/**
+ * The options passed to {@link TaskStore.recoverTasks}
+ * @public
+ */
+export type TaskStoreRecoverTaskOptions = {
+  timeoutS: number;
```

Comment:
Think we should make this `timeout: HumanDuration`.

Looks like this:

`timeout: { seconds: 100 }` in practice